orderer/consensus/etcdraft/node.go

```diff
@@ -276,6 +276,16 @@ func (n *node) abdicateLeader(currentLead uint64) {
 	timer := n.clock.NewTimer(time.Duration(n.config.ElectionTick) * n.tickInterval)
 	defer timer.Stop() // prevent timer leak
 
+	// Revalidate the status before waiting on notifyc channel as Ready go routine
```

Comment:
One thing I do not understand is, why do we even wait for the leadership to be transferred before configuring the communication framework, if we're a follower? 

The one that is doing the leadership transfer is the leader. The followers have no part in this.